spec/policies/form_policy_spec.rb

```diff
@@ -64,6 +66,17 @@
 
     let(:gds_forms) { build_list :form, 2, org: "gds" }
 
+    it "uses organisation slug to scope what forms a user can see" do
```

Comment:
This feels like we are testing implementation rather than behaviour that we are expecting from the scope policy.  

We shouldn't care how we get a list of forms owned by the user but more of a case of "only return a list of forms the user should be able to see.

Reply:
The reason for this test was to catch a very specific issue; I forgot to change `user.organisation_slug` in the policy scope to `user.organisation.slug`, and spent ages trying to figure it out because it wasn't super obvious. I added feature tests in https://github.com/alphagov/forms-admin/pull/414/files#diff-7765f51619f0b6c7cb0ecacea452722004fcbc8bca758bdc4b35ef64614c476aR37 to test the behaviour (and they helped me track down the issue), but I wanted to make sure the test was very clear on what the implementation should be, so it would be clear if someone accidentally changed the code what was going on.